tests/core/test_examples.py

```diff
@@ -100,13 +86,16 @@ def loop():
         assert responses[0]["text"] == "chitchat"
 
 
-async def test_concertbot_training():
-    from examples.concertbot.train import train_dialogue
+async def test_restaurantbot_example():
+    sys.path.append("examples/restaurantbot/")
+    from run import train_core, parse
 
-    r = await train_dialogue(
-        domain_file="examples/concertbot/domain.yml",
-        stories_file="examples/concertbot/data/stories.md",
-        model_path="examples/concertbot/models/dialogue",
-        policy_config="examples/concertbot/config.yml",
+    p = "examples/restaurantbot/"
+    stories = os.path.join("data", "test_stories", "stories_babi_small.md")
+    core_model_path = await train_core(
+        os.path.join(p, "domain.yml"), os.path.join(p, "models", "core"), stories
     )
-    assert r is not None
+
+    responses = await parse("/greet", core_model_path)
```

Comment:
sweet, would be good if we could include the training of NLU in here and then parse "hello" (something in the nlu data file) to make sure both are working